data/samplesheets/190719_SN737_0469_BCDA1CACXX_spike.csv

```diff
@@ -0,0 +1,57 @@
+[Header],,,,,,,,,,,
+IEMFileVersion,5,,,,,,,,,,
+Date,07/17/19,,,,,,,,,,
+Workflow,GenerateFASTQ,,,,,,,,,,
+Application,HiSeq FASTQ Only,,,,,,,,,,
+Instrument Type,HiSeq 1500/2500,,,,,,,,,,
+Assay,Agilent_Sure_SelectXT,,,,,,,,,,
+Index Adapters,Agilent_Sure_SelectXT,,,,,,,,,,
+Description,PE-Amp-Lin-Block-Hyb-v 8.0,,,,,,,,,,
+Chemistry,default,,,,,,,,,,
+Operator,Daniel Scholtyssik / Katayoun Alemazkur / Frauke Meyer,,,,,,,,,,
+Investigator,Marina Oldenburg / Marc Remke,,,,,,,,,,
+Backup_NAS_name,Ingerimm,,,,,,,,,,
+kind_of_run,2x101bp,,,,,,,,,,
+[Reads],,,,,,,,,,,
+101,,,,,,,,,,,
+101,,,,,,,,,,,
+,,,,,,,,,,,
+[Settings],,,,,,,,,,,
+ReverseComplement,0,,,,,,,,,,
+Adapter,AGATCGGAAGAGCACACGTCTGAACTCCAGTCA,,,,,,,,,,
+AdapterRead2,AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT,,,,,,,,,,
+,,,,,,,,,,,
+[Data],,,,,,,,,,,
+Lane,Sample_ID,Sample_Name,Sample_Plate,Sample_Well,I7_Index_ID,index,Sample_Project,Description,spike_entity_id,spike_entity_role,spike_ignore_sample
+1,KB0252_f,,,,D01,CACTTCGA,Keimbahn,set1,KB0252,father,
+1,KB0252_m,,,,E01,GCCAAGAC,Keimbahn,set1,KB0252,mother,
+1,ALPS_73,,,,G01,ATTGGCTC,Alps,set1,ALPS_73,child,
+1,ALPS_74,,,,H01,GATGAATC,Alps,set1,ALPS_74,child,
+2,KB0252_f,,,,D01,CACTTCGA,Keimbahn,set1,KB0252,father,
+2,KB0252_m,,,,E01,GCCAAGAC,Keimbahn,set1,KB0252,mother,
+2,mRuxo_394_N,,,,A02,AGCAGGAA,Maus_Hauer,set2,mruxo_394,healthy,
```

Comment:
please synchronise capitalization, i.e. if the sample is named `mRuxo_394_N` you should also call the spike_entity_name `mRuxo_394` not `mruxo_394` - but this is only a stylistic issue.